package.json

```diff
@@ -261,6 +262,75 @@
             }
           }
         ]
+      },
+      {
+        "type": "docker-netcoreapp",
+        "label": "Docker: Launch .NET Core",
+        "configurationSnippets": [
+          {
+            "label": "Docker: Launch .NET Core",
+            "description": "Docker: Launch .NET Core",
+            "body": {
+              "name": "Docker: Launch .NET Core",
+              "type": "docker-netcoreapp",
+              "request": "launch",
+              "preLaunchTask": "build"
+            }
+          }
+        ],
+        "configurationAttributes": {
+          "launch": {
+            "properties": {
+              "dockerOptions": {
```

Comment:
When transforming the `docker-coreclr` debug configuration into the debug configuration for `coreclr`, some of the configuration is merged (e.g. the `launchBrowser` property) while some is overwritten (e.g. the `program` and `cwd` properties).  Because of the merge, and because the debug configurations don't seem to be rigorously defined, I was concerned about collisions with new Docker-related properties, so made the properties hang off of a single, Docker-specific property.

I'll admit to being overly paranoid (for the first PR in a new codebase).  I've updated the configuration such that the properties now hang off of the root, and updated the `build` and `run` properties to `dockerBuild` and `dockerRun`, respectively, to avoid any confusion about the local .NET Core build/run.